test/python/fx_importer/v2.3/mutation_import.py

```diff
@@ -105,6 +105,27 @@ def forward(self, x):
     m.operation.verify()
 
 
+@run
+# CHECK-LABEL: test_frozen_buffer_non_persistent
+# CHECK: %[[buffer_literal:.+]] = torch.vtensor.literal
+# CHECK: %[[mul:.+]] = torch.aten.mul.Tensor %arg0, %0
+# CHECK: return %[[mul]]
+def test_frozen_buffer_non_persistent():
+    class Basic(nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.register_buffer("buffer", torch.randn(3, 4), persistent=False)
```

Comment:
Can you think of any way to test persistent=True? Looks like that code path doesn't get tested.

Reply:
When `register_buffer`, the `persistent` is by default set to be true. `register_buffer(name, tensor, persistent=True)`. So the `test_frozen_buffer()` test that run before the added `test_frozen_buffer_non_persistent()` already covered the `persistent=True` case. But if you prefer, I can make `persistent=True` explicit in the `test_frozen_buffer()` test.

Reply:
Oh, thanks. That works then. I wasn't looking at all of the tests in context.